src/js/dataMapping/search/awardsOperationKeys.js

```diff
@@ -33,6 +33,7 @@ export const timePeriodKeys = {
 export const agencyKeys = {
     type: 'type',
     tier: 'tier',
+    toptier_name: 'toptier_name',
```

Comment:
For consistency, we could use `toptierName` for the property, which maps to the snake case API value (prefer camel case in the JS).